test/ui-automation/test/helpers/wallet.js

```diff
@@ -95,9 +128,10 @@ async function _getThirdPartyLogin(email) {
     return true;
   });
 
-  const signInButton = await $('#accept');
-  await signInButton.click();
+  const loginInButton = await $('#accept');
+  await loginInButton.click();
```

Comment:
Maybe you meant `logIn`? In either way, aren't we using `signIn`/`signUp` syntax now?

Reply:
yeah, makes sense. 

cc: @talwinder50

Reply:
This loginin button is for login page which comes in popup we can rename to popupsignin may be

Reply:
[renamed](https://github.com/trustbloc/wallet/compare/4a4426f351c82128fb4613e2c3aa0441bd1f1f28..74baac2b14e8c73755884b7e078dd88f7246bbb2) to `oidcLoginButton